oligo_designer_toolsuite/IO/_database.py

```diff
@@ -383,9 +355,10 @@ def __init__(
 
         genome_assembly = "GRCh38"
```

Comment:
genome assembly should not be harcoded, since it depends on the species and annotation release and should be returned correctly